book_src/Operating System From 0 to 1.lyx

```diff
@@ -2643,6 +2644,10 @@ status collapsed
 \end_inset
 
 
+\begin_inset Separator latexpar

```

Comment:
It seems there are some incompatibilities between Lyx 2.1 and 2.2.  Could you stage only the edited text and not the whole file?